### PR TITLE
Push docker

### PR DESCRIPTION
**Pull request by miklosduma**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+RUN apt-get update \
+ && apt-get upgrade -y \
+ && apt-get install -y --no-install-recommends \
+    build-essential \
+ && rm -rf /var/lib/apt/lists/*
+
+COPY --from=builder /home/builder/workdir/dist ./sse_cli/dist
+RUN pip3 install sse_cli/dist/sparkl_cli-*.tar.gz
+RUN rm -rf sse_cli
```
Comment (lucafavatellawork) [inline docker/Dockerfile:62]: Not setting non-root `USER`?

Comment (lucafavatellawork) [inline docker/Dockerfile:62]: I may be missing something, though I do not see `CMD` / `ENTRYPOINT` / intended usage.

My preference is to see smoke test usage of of the Docker image in the CI (make target?) though if that is considered too much at least a line in the README.

My main concern is that the CLI makes stateful (filesystem-based session) an API that is in each request self-contained (HTTP as exposed by SSE instance) so I am not sure how this is meant to handle filesystem state, in particular in sequence of commands.

Comment (miklosduma) [reply docker/Dockerfile]: Addressed in commit.

Comment (miklosduma) [reply docker/Dockerfile]: I added an entrypoint and a default command. It is also called in the CI to see if it works. Could you please have a second look? Thanks!

File: .travis.yml [hunk 2]
```diff
@@ -21,20 +21,51 @@ aliases:
...
+      language: minimal
+      git:
+        depth: false
+      services:
+        - docker
+      script:
+        - make -C docker docker
+        - make -C docker latest_docker
+        - docker tag sparkl/cli:latest sparkl/cli:"${TRAVIS_BRANCH:?}"
+        - docker run -it sparkl/cli:$latest
```
Comment (lucafavatellawork) [inline .travis.yml:63]: Is the `$` on purpose?

File: .travis.yml [hunk 3]
```diff
@@ -21,20 +21,51 @@ aliases:
...
+      before_deploy:
+        - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
```
Comment (lucafavatellawork) [inline .travis.yml:65]: Could you please confirm these two variables are set in https://travis-ci.com/github/opensparkl/sse_cli/settings as secrets?

File: .travis.yml (hunk 2 above)
Comment (miklosduma) [reply .travis.yml]: Nope, typo, nice catch!

**Pull request merged**